src/linux/backend-v4l2.cpp

```diff
@@ -520,7 +520,7 @@ namespace librealsense
         void v4l_uvc_device::probe_and_commit( stream_profile profile, bool zero_copy,  frame_callback callback, int buffers)
         {
             if(!zero_copy)
-                buffers = 1;
+                buffers = 3;
```

Comment:
was and is a magic number, please use come `const int <name> = 3` instead

Reply:
Itay remove this in his PR - #810